src/DataDrivenDiffEq.jl

```diff
@@ -7,88 +7,164 @@ using DocStringExtensions
 using LinearAlgebra
 using DiffEqBase
 using ModelingToolkit
+
 using QuadGK
 using Statistics
-using DSP
-using FiniteDifferences, DataInterpolations
+#using DSP
+using DataInterpolations
+
+using Reexport
 using Compat
 using DocStringExtensions
 
-abstract type AbstractKoopmanOperator <: Function end;
 
-include("./optimizers/Optimize.jl")
-using .Optimize
+@reexport using ModelingToolkit: states, parameters, independent_variable, observed, controls
```

Comment:
Okay, after removing the `.julia` folder I thought I would encounter an error here, since the precompilation error of @YingboMa pointed to this line. However, it works fine for me, so the reexport and the dispatch of `ModelingToolkit.controls(b::AbstractBasis)`in `basis.jl` should not be the problem, which I previously suspected.

Reply:
I will double check this on a different machine